src/components/button/index.tsx

```diff
@@ -18,6 +18,7 @@ const Button = ({
   size = 'medium',
   text,
   type = 'button',
+  form,
```

Comment:
🤔  We could consider spreading props here so we don't need to add native button attributes to this component